src/array.js

```diff
@@ -89,7 +104,9 @@ function getStringsLength(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-	throw new Error("Not implemented");
+	let sum=0;
+	arr.forEach(elmt => sum += elmt);
+	return sum;
```

Comment:
Great job using `forEach`. Do try implementing it with `.reduce`

Reply:
Learnt a new method thanks bhaiya.
Aise hi hai na bhaiya:?
function getItemsSum(arr) {
	return arr.reduce((accSum,elmt) => accSum + elmt,0);
}

Reply:
Yes. You can update the code in a new commit.